src/components/Text/Text.tsx

```diff
@@ -1,37 +1,31 @@
 import * as React from 'react';
 import { Text as ChakraText, BoxProps } from '@chakra-ui/core';
 
-export enum TextVariants {
-  CONTENT = 'CONTENT',
-  CONTENTBOLD = 'CONTENTBOLD',
-  SUBCONTENT = 'SUBCONTENT',
-  SUBCONTENTBOLD = 'SUBCONTENTBOLD',
+export enum TextWeights {
+  normal = 'normal',
```

Comment:
I'd add italic here too.

Reply:
It's not a weight, and not part of the specs IIRC

Reply:
Right 🤦 I always forget that haha